service/history/handler.go

```diff
@@ -637,7 +637,18 @@ func (h *Handler) ExecuteMultiOperation(
 	if err != nil {
 		return nil, h.convertError(err)
 	}
-	// TODO: shardContext.NewVectorClock()
+
+	for _, opResp := range response.Responses {
+		if startResp := opResp.GetStartWorkflow(); startResp != nil {
+			if startResp.Clock == nil {
+				startResp.Clock, err = shardContext.NewVectorClock()
```

Comment:
This step needs to grab shard lock, so I'd say just new the clock once and use it in all the responses.

Reply:
There will never be more than one Start, but I can still do that 👍